### ISSUE TITLE
Game crashes when i skip time past event.

### ISSUE THREAD

**Issue by QuandalePr1ngle** [JordanSantiagoYT/FNF-PsychEngine-NoBotplayLag#8]
### Describe your bug here.

Every time when i try to skip time past event the engine crashes (in charting mode)

The script itself:

function onCreatePost()
	makeLuaSprite('darkerbg', '', 0, 0)
	setScrollFactor('darkerbg', 0, 0)
	makeGraphic('darkerbg', 3840, 2160, '000000')
	addLuaSprite('darkerbg', false)
	setProperty('darkerbg.alpha', 0)
	screenCenter('darkerbg', 'xy')
end
function onEvent(n, v1, v2)
	if n == 'darkbg' then
		doTweenAlpha('Buh', 'darkerbg', v1, v2, 'linear')
	end
	if n == 'darkbg' then
		doTweenAlpha('Duh', 'darkerbg', v1, v2, 'linear')
	end
end

### Command Prompt/Terminal logs (if existing)

```bash
![Безіменний](https://github.com/JordanSantiagoYT/FNF-PsychEngine-NoBotplayLag/assets/133515732/302bcb1d-537d-4130-a624-18f38c63ddbf)
```

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

No

**Comment by QuandalePr1ngle** [comment]
ah photo didnt load properly
![Безіменний](https://github.com/JordanSantiagoYT/FNF-PsychEngine-NoBotplayLag/assets/133515732/06ddbdab-d348-42ee-8fc6-7da721d58889)

**Comment by JordanSantiagoYT** [comment]
i think ive found the issue, apparently resizeByRatio crashes the game if you place a "Change Scroll Speed" event, and then skip time to after the event happens. it should be fixed with 1.8.0's release

**Comment by QuandalePr1ngle** [comment]
Still have same problem, but now it crashes whanever im using skipping time even before events.

**Comment by JordanSantiagoYT** [comment]
i dunno then, it doesn't crash for me

https://github.com/JordanSantiagoYT/FNF-PsychEngine-NoBotplayLag/assets/61796448/aea3e127-ca72-40fd-883a-894a5e9bea3a

**Comment by QuandalePr1ngle** [comment]
i think you didnt get me


https://github.com/JordanSantiagoYT/FNF-PsychEngine-NoBotplayLag/assets/133515732/aa011a48-0ceb-4ca1-b0e6-90ae2110d336

it crashes right when event starts.

**Comment by JordanSantiagoYT** [comment]
is this in 1.8.0?

**Comment by QuandalePr1ngle** [comment]
found out that some songs work properly and some doesn't, idk why

**Comment by JordanSantiagoYT** [comment]
should be fixed in 1.9.0

**Comment by QuandalePr1ngle** [comment]
Finally
